docs/sphinx-docs/source/user/RELEASE.rst

```diff
@@ -13,6 +13,151 @@ Release Notes
 Features
 ========
 
+New in Version 6.0.0
+--------------------
+This is a major release with a number of new features, enhancements, and bug fixes as described below. Many of the new
+features are now highlighted in our 'What's New' window displayed when SasView starts.
```

Comment:
Going from v5 to v6 means more than feature additions. In principle breaking changes? In particular moving sasdata out of sasview and moving from PyQt to Pyside for the GUI package? Seems like it should be made clear somehow why this is not 5.1 (a feature release)?

Also - for a future PR probably but I would recommend highlighting at the top some of the big changes for users like a proper settings option and the combined orders of magnitude speed increases in both reading the PDB and then calculating the scattering from that PDB? Or maybe we just leave those now in "What's new" window?